acs/binary_map.go

```diff
@@ -0,0 +1,17 @@
+package acs
+
+import (
+	"sync"
+
+	"github.com/DE-labtory/cleisthenes"
+)
+
+type binaryStateMap struct {
+	lock  sync.RWMutex
+	items map[cleisthenes.Address]cleisthenes.BinaryState
+}
+
+type broadcastDataMap struct {
```

Comment:
How about `bbaStateMap` or `rbcOutputMap` rather than `binaryStateMap`, `broadcastDataMap`? 
`acs` looks at only `bba` and `rbc` itself, not the properties of component

Reply:
No I think it is better to have much general name.